internal/provider/provider.go

```diff
@@ -534,7 +541,7 @@ func populateIsNull(resourceData *schema.ResourceData) (result interface{}, err
 
 // valueAsString takes a cty.Value that may be a string or null, and converts it to either a Go string
```

Comment:
This comment was wrong to begin with 😞 Might as well fix it now:

```suggestion
// valueAsString takes a cty.Value that may be a string or null, and converts it to a Go string,
// which will be empty if the input value was null.
```